frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelDocs.tsx

```diff
@@ -1,13 +1,48 @@
-import { useValues } from 'kea'
-import { sidePanelDocsLogic } from './sidePanelDocsLogic'
+import { useActions, useValues } from 'kea'
+import { POSTHOG_WEBSITE_ORIGIN, sidePanelDocsLogic } from './sidePanelDocsLogic'
+import { useEffect, useRef } from 'react'
+import clsx from 'clsx'
+import { SidePanelPaneHeader } from '../components/SidePanelPane'
+import { LemonButton } from '@posthog/lemon-ui'
+import { IconExternal } from '@posthog/icons'
 
 export const SidePanelDocs = (): JSX.Element => {
-    const { path } = useValues(sidePanelDocsLogic)
+    const { iframeSrc, currentUrl } = useValues(sidePanelDocsLogic)
+    const { updatePath, unmountIframe, closeSidePanel } = useActions(sidePanelDocsLogic)
+    const ref = useRef<HTMLIFrameElement>(null)
+
+    useEffect(() => {
+        const onMessage = (event: MessageEvent): void => {
+            if (event.origin === POSTHOG_WEBSITE_ORIGIN) {
+                if (typeof event.data === 'string') {
+                    updatePath(event.data)
```

Comment:
If possible we change the format on the website side so that data is an object instead